samples/client/petstore/swift3/default/PetstoreClient/Classes/Swaggers/APIs/FakeAPI.swift

```diff
@@ -208,9 +208,9 @@ open class FakeAPI: APIBase {
 
         let url = NSURLComponents(string: URLString)
         url?.queryItems = APIHelper.mapValuesToQueryItems(values:[
-                "enumQueryStringArray": enumQueryStringArray, 
-                "enumQueryString": enumQueryString, 
-                "enumQueryInteger": enumQueryInteger
+            "enum_query_string_array": enumQueryStringArray, 
+            "enum_query_string": enumQueryString?.rawValue, 
+            "enum_query_integer": enumQueryInteger?.encodeToJSON()
```

Comment:
@tomekc @wing328 is this correct for the integer value - was this broken as well?  I notice the double above in the formParams wasn't encoded...